Makefile

```diff
@@ -72,6 +88,29 @@ unit:
 integration:
 	poetry run pytest tests/integration --axe-version 4.9.1 --chromedriver-path $$(which chromedriver)
 
+
+# Install project dependencies for GitHub Actions
+gha_install_project: install_python_deps install_npm_deps
```

Comment:
I suggest we remove these extra tasks that only run on github actions - I don't think they are gaining us anything.

I can't remember what we discussed in standup, but I understood the point of this ticket to be about removing duplication between local env setup and CI setup. If it's not easy to do that I think it's fine to leave stuff as it is.